Pocket.Logger/For.MicrosoftExtensionsLogging/Tests/When_publishing_from_ILogger_to_PocketLogger.cs

```diff
@@ -16,8 +16,7 @@ public class When_publishing_from_ILogger_to_PocketLogger : IDisposable
 
         public When_publishing_from_ILogger_to_PocketLogger(ITestOutputHelper output)
         {
-            loggerFactory = new LoggerFactory()
-                .AddConsole()
+            loggerFactory = LoggerFactory.Create(builder => builder.AddConsole())
```

Comment:
`AddConsole` extension method is on `ILoggingBuilder` instead of `ILoggerFactory`:
```c#
namespace Microsoft.Extensions.Logging
{
    public static class ConsoleLoggerExtensions
    {
        public static ILoggingBuilder AddConsole(this ILoggingBuilder builder)
        {
        }
    }
}
```